.travis.yml

```diff
@@ -2,180 +2,183 @@ language: python
-    - mkdir -p local_configs/routinator/tals
-    - sudo chown -R 1012:1012 local_configs/routinator/tals
-    - mkdir -p local_configs/routinator/rpki-repo
-    - sudo chown -R 1012:1012 local_configs/routinator/rpki-repo
-    - cp other/routinator/routinator.conf local_configs/routinator/routinator.conf
-    - sudo chown -R 1012:1012 local_configs/routinator/routinator.conf
-    - docker build -t artemis_rpki_testing testing/rpki
-    - docker run --rm -v $(pwd)/local_configs/routinator/tals:/home/routinator/.rpki-cache/tals nlnetlabs/routinator init -f --accept-arin-rpa
-    - docker-compose -f docker-compose.testrpki.yaml -f docker-compose.routinator.yaml up -d
-    script:
-    - docker run --rm -ti --env-file .env --network artemis_artemis artemis_rpki_testing
-    after_success:
-    - if [ ! $IS_FORK ]; then sh convert_coverage.sh; fi
-    - if [ ! $IS_FORK ]; then codecov; fi
-    after_failure:
-    - docker-compose -f docker-compose.testrpki.yaml logs
-  - name: ZAP Baseline (Passive)
-    before_script:
-    - docker pull ictu/zap2docker-weekly
-    - if [ ! $IS_FORK ]; then docker pull inspiregroup/artemis-backend:${DOCKER_TAG}; fi
-    - if [ ! $IS_FORK ]; then docker pull inspiregroup/artemis-frontend:${DOCKER_TAG}; fi
-    - if [ ! $IS_FORK ]; then docker pull inspiregroup/artemis-monitor:${DOCKER_TAG}; fi
-    - if [ ! $IS_FORK ]; then docker tag inspiregroup/artemis-backend:${DOCKER_TAG} artemis_backend:latest; fi
-    - if [ ! $IS_FORK ]; then docker tag inspiregroup/artemis-frontend:${DOCKER_TAG} artemis_frontend:latest; fi
-    - if [ ! $IS_FORK ]; then docker tag inspiregroup/artemis-monitor:${DOCKER_TAG} artemis_monitor:latest; fi
-    - if [ $IS_FORK ]; then sed -i "s/image\:\sinspiregroup\/artemis-backend:\${SYSTEM_VERSION}/image\:\ artemis_backend\\n        build\:\ \.\/backend\//g" docker-compose.yaml; fi
-    - if [ $IS_FORK ]; then sed -i "s/image\:\sinspiregroup\/artemis-frontend:\${SYSTEM_VERSION}/image\:\ artemis_frontend\\n        build\:\ \.\/frontend\//g" docker-compose.yaml; fi
-    - if [ $IS_FORK ]; then sed -i "s/image\:\sinspiregroup\/artemis-monitor:\${SYSTEM_VERSION}/image\:\ artemis_monitor\\n        build\:\ \.\/monitor\//g" docker-compose.yaml; fi
-    - docker-compose up -d
-    script:
-    - sleep 15
-    - docker run -t --network artemis_artemis ictu/zap2docker-weekly zap-baseline.py -t https://nginx --hook=/zap/auth_hook.py -z 'auth.loginurl=https://nginx/login auth.username="admin" auth.password="admin123" auth.username_field="email" auth.password_field="password" auth.submit_field="submit" auth.exclude=".*logout.*,\Qhttps://nginx/logout\E.*"' || true
+    - stage: checks
+      name: PreCommitCheck
+      install:
+        - python -m pip install pre-commit pytest pytest-cov codecov
+        - python -m pip install -r backend/requirements.txt
+        - pre-commit install
+      script:
+        - echo $TRAVIS_PULL_REQUEST_SLUG
+        - SKIP=unittest pre-commit run --all-files
```

Comment:
These next 3 lines is the actual difference.
I reformatted and it shows everything as a diff

Reply:
no problem. Approved.